apps/designer/app/canvas/features/text-editor/text-editor.tsx

```diff
@@ -10,6 +11,21 @@ import type { ChildrenUpdates, Instance } from "@webstudio-is/react-sdk";
 import { ToolbarConnectorPlugin } from "./toolbar-connector";
 import { type Refs, $convertToLexical, $convertToUpdates } from "./interop";
 
+const BindInstanceToNodePlugin = ({ refs }: { refs: Refs }) => {
+  const [editor] = useLexicalComposerContext();
+  useEffect(() => {
+    for (const [nodeKey, instance] of refs) {
+      // extract key from stored key:style format
+      const [key] = nodeKey.split(":");
+      const element = editor.getElementByKey(key);
+      if (element) {
+        element.setAttribute("data-ws-id", instance.id);
```

Comment:
lets use idAttribute constant I made reusable from a package? thanks for fixing it, didn't realize nodes won't have the attribute